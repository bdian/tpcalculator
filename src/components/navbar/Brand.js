import React from 'react';
import styled from 'styled-components';

const Brand = () => {
  return (
    <Image src="../../assets/logo-min.png" alt="TP-Calculator Logo" />
  );
};

export default Brand;

const Image = styled.img`
  height: 85%;
  margin: auto 0;
`;
