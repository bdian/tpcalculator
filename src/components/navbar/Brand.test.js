import React from 'react';
import Brand from './Brand';
import renderer from 'react-test-renderer';

test('Brand snapshot', () => {
  const component = renderer.create(
    <Brand />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});