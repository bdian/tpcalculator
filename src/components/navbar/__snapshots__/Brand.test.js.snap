// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brand snapshot 1`] = `
<img
  alt="TP-Calculator Logo"
  className="sc-AxjAm kzuEgZ"
  src="../../assets/logo-min.png"
/>
`;
